### PR TITLE
Update BitsServe Policy IDs

### PR DESCRIPTION
**Pull request by yopkub**
Update PolicyIDs

### CODE REVIEW

File: Update BitsServe PolidyIDs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "project": "BitsServe",
+    "tags": [
+        "BitsServe"
+    ],
+    "policies": [
+        "cc8e45713c50e0a58512b97e2bd223000a21c798e526f224f5eea30a",
+        "4b97c699e870afdfdcde4b666f777ec2e8c599fd1a8f12a01e50da91"
```
Comment (yopkub) [inline Update BitsServe PolidyIDs:8]: Addition PolicyIDs

**Comment by yopkub** [comment]
@Dean00K 
Hello, Thank you for always support.

This first time to update PolicyIDs,
I don't sure I create code correctly ? because I see other creator pull request update at code file will be have some "red hilight" but I havn't.

If have any request me, please let me know ASAP.
Thank you.

**Comment by Dean00K** [comment]
All good, the other lines on other peoples commits are red because they edit and add in single lines and remove single lines. looks like you just copy and pasted over the whole thing again? It's fine, still works.

**Comment by yopkub** [comment]
@Dean00K Thank you very much. Bro

**Pull request merged**